### ISSUE TITLE
Allow multiple chains to be run

### ISSUE THREAD

**Issue by mhtess** [mhtess/rwebppl#10]
Bonus if this could be done in parallel.

Minimally, this will involve running multiple `system` calls, outputting the data to different temp files, and reading and parsing the temp files, appending the chain #

**Comment by mhtess** [comment]
Per our discussion today, we could do this potentially by having the rwebppl js take the temp_file output filename as input argument. Then we could wrap the rwebppl R function within a for loop or a paralleled version, passing a new filename for each chain.  We could concatenate the output so that an ERP becomes a data frame that looks like:

```
support | probs | chain
-------------------------
a | 0.25 | 1
...
a | 0.23 | 2
...
```

The halting conditions should probably be by default to have the program halt if one of the chains errors out. It would be nice to have a `force=T` option to change the halting condition to wait for each chain to either error or complete, and then concatenate the output for those chains that completed.
